ignite/handlers/checkpoint.py

```diff
@@ -837,7 +850,7 @@ def __init__(
         )
 
     @property
-    def last_checkpoint(self) -> Union[str, None]:
+    def last_checkpoint(self) -> Optional[str]:
```

Comment:
I was wrong here. Output is `Optional[Path]`. 
By the way, we have to make this uniform between `ModelCheckpoint` and `Checkpoint`

Reply:
```suggestion
    def last_checkpoint(self) -> Optional[Path]:
```